### ISSUE TITLE
Error while compiling from the source ... undefined: sqlite3.Error

### ISSUE THREAD

**Issue by chosenonehacks** [gophish/gophish#2771]
What version of Gophish are you using?: latest public github repository from here: https://github.com/gophish/gophish 

Brief description of the issue: Wen trying to compile from this github repo I am getting compilation error.

What are you expecting to see happen? : get compiled version

What are you seeing happen? error when compiling:

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below: Nope, it's is not.

Please provide as many steps as you can to reproduce the problem:
Step 1. git clone https://github.com/gophish/gophish 
Step 2. go build (which gets error as below):
```
# bitbucket.org/liamstask/goose/lib/goose
../go/pkg/mod/bitbucket.org/liamstask/goose@v0.0.0-20150115234039-8488cc47d90c/lib/goose/dialect.go:119:15: undefined: sqlite3.Error
```
 

Additional info:
Go version: go1.20.1 linux/amd64
OS Ubuntu 20.04 LTS x64 Fresh AWS LightSail OS only box

**Comment by Eonevsg** [comment]
Have the same problem, created an issue yesterday https://github.com/gophish/gophish/issues/2768

**Comment by glennzw** [comment]
Solution in #2768 

tl;dr: `apt install gcc`

**Comment by chosenonehacks** [comment]
thank you all, that worked

**Comment by SirDanje** [comment]
Still not working for me
